.github/workflows/test.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       with:
         node-version: ${{ matrix.node-version }}
     - run: npm install
-    - run: npm start & sleep 5 && npm test
+    - run: npm test
```

Comment:
In the future - Do not add extra files to PR
To add the file to a commit use git add src/.